api/seedreconfig/seedreconfig.go

```diff
@@ -0,0 +1,86 @@
+package seedreconfig
+
+type PEM string
+
+const (
+	SeedReconfigVersion = 1
+)
+
+// SeedReconfiguration contains all the information that is required to
+// transform a machine started from a SNO seed image (which contains dummy seed
+// configuration) into a SNO cluster with a desired configuration. During an
+// IBU, this information is taken from the cluster that is being upgraded (the
+// original SNO's LCA writes a file to the seed stateroot to communicate this
+// information). During an IBI, this information is placed in the configuration
+// ISO created by the cluster-relocation-operator.
+//
+// WARNING: Changes to this struct and its sub-structs should not be made
+// lightly, as it is also used by the cluster-relocation-operator. Any changes
+// made here will also need to be handled in the cluster-relocation-operator or
+// be backwards compatible. If you've made a breaking change, you will need to
+// increment the SeedReconfigVersion constant to avoid silently breakage and
+// allow for backwards compatibility code.
+type SeedReconfiguration struct {
+	// The version of the SeedReconfiguration struct format. This is used to detect
+	// breaking changes to the struct.
+	APIVersion int `json:"api_version"`
+
+	// The desired base domain for the cluster. Equivalent to install-config.yaml's baseDomain.
+	// This will replace the base domain of the seed cluster.
+	BaseDomain string `json:"base_domain,omitempty"`
+
+	// The desired cluster name for the cluster. Equivalent to install-config.yaml's clusterName.
+	// This will replace the cluster name of the seed cluster.
+	ClusterName string `json:"cluster_name,omitempty"`
+
+	// The desired cluster-ID. During an IBU, this is the ID of the original
+	// SNO. During an IBI, this can either be empty, in which case a new
+	// cluster-ID will be generated by LCA, or it can be set to the ID of the
+	// new cluster, in case one had to be pre-generated for some reason.
+	ClusterID string `json:"cluster_id,omitempty"`
+
+	// The desired IP address of the SNO node.
+	NodeIP string `json:"node_ip,omitempty"`
+
+	// ?
+	ReleaseRegistry string `json:"release_registry,omitempty"`
```

Comment:
https://github.com/openshift/image-based-install-operator/pull/26#discussion_r1452510794

Reply:
The release registry for the cluster version release image